app/views/questions/index.erb

```diff
@@ -5,12 +5,12 @@
 <% end %>
 </div>
 
-<div class="container">
+<div class="container collection">
   <% if @questions %>
     <ul>
     <% @questions.each do |question| %>
-      <li><a href="/questions/<%=question.id%>"><%= question.title %></a></li>
+      <li><a class="collection-item" href="/questions/<%=question.id%>"><span class="badge">4 Votes</span><span class="badge">4 Answer(s)</span><%= question.title %></a></li>
```

Comment:
I haven't put in the ruby code to add up the votes and comments yet, seeing if this design decision works for everyone